### PR TITLE
Fixed WalletConnectQRImage class multiple handler bug on enable/disable.

### PR DESCRIPTION
**Pull request by AlLansley**
- Modified QR code image size to be smaller & hence consume less texture memory,
- Made QR code texture use point / nearest filtering to preserve sharp edges of pixel blocks,
- Added comments.

### CODE REVIEW

**Comment by gigajuwels** [comment]
@ecp4224 please review

File: Assets/WalletConnectUnity/Scripts/WalletConnectSharp.Unity/UI/WalletConnectQRImage.cs [hunk 1]
```diff
@@ -26,23 +37,47 @@ void OnEnable()
             return;
         }
         
-        walletConnect.ConnectionStarted += WalletConnectOnConnectionStarted;
+        // Only register the WalletConnectOnConnectionStarted handler once
+        if (!registeredOnConnectionStartEvent)
+        {
+            registeredOnConnectionStartEvent = true;
+            walletConnect.ConnectionStarted += WalletConnectOnConnectionStarted;
```
Comment (ecp4224) [inline Assets/WalletConnectUnity/Scripts/WalletConnectSharp.Unity/UI/WalletConnectQRImage.cs:44]: I guess we should be using the new `walletConnect.NewSessionStarted` event handler to only trigger a QRCode update when a new session is created. 

Having the bool flag check here is still needed so we dont subscribe more than once

**Review by ecp4224 — CHANGES_REQUESTED** [review]
Everything LGTM, just one small change to use new events

File: Assets/WalletConnectUnity/Scripts/WalletConnectSharp.Unity/UI/WalletConnectQRImage.cs (hunk 1 above)
Comment (ecp4224) [reply Assets/WalletConnectUnity/Scripts/WalletConnectSharp.Unity/UI/WalletConnectQRImage.cs:44]: Actually never mind, we want the QR Code to generate when we connect to the bridge, not after the session gets created/connected

**Review by ecp4224 — APPROVED** [review]

**Pull request merged**